game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadFileDescription.java

```diff
@@ -23,7 +31,17 @@
   enum DownloadType {
     MAP,
     MAP_SKIN,
-    MAP_TOOL
+    MAP_TOOL;
+
+    static DownloadType fromString(final String type) {
```

Comment:
Hmm I mainly wanted to suggest documenting that the method returns "MAP" for unknown values instead of throwing an exception, not sure if this sort of API docuentation makes sense for private methods, but I still think it would be nice nevertheless

Reply:
I'm not sure if it'd really just be faster to read the method to see it does not throw. 
FWIW, this method is updated soon and eventually goes away completely in a future revision.